src/rules.scala

```diff
@@ -360,12 +370,15 @@ object Plugin extends sbt.Plugin {
     useSdkProguard          <<= proguardScala (!_),
     useProguardInDebug      <<= proguardScala,
     collectResources        <<= collectResourcesTaskDef,
+    collectResources        <<= collectResources dependsOn renderscript,
     packageResources        <<= packageResourcesTaskDef,
     apkFile                 <<= (name, projectLayout) { (n,l) =>
       val apkdir = l.bin / ".build_integration"
       apkdir.mkdirs()
       apkdir / (n + "-BUILD-INTEGRATION.apk")
     },
+    collectProjectJni       <<= collectProjectJniTaskDef,
+    collectProjectJni       <<= collectProjectJni dependsOn renderscript,
     collectJni              <<= collectJniTaskDef,
```

Comment:
Nope, not sure what you mean exactly, but I didn't remove any tasks, only added couple more.
renderscript is still there, collectJni is now separated in two, collectProjectJni and collectJni (which uses results of the previous one)